internal/manifests/collector/rbac.go

```diff
@@ -17,6 +17,11 @@ import (
 )
 
 func ClusterRole(params manifests.Params) (*rbacv1.ClusterRole, error) {
+	// Skip RBAC creation if custom ServiceAccount is provided
```

Comment:
Maybe it makes more sense to check if the service account already has permission for the needed resources... what do you think?